src/hsm.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * #%L
+ * %%
+ * Copyright (C) 2022 BMW Car IT GmbH
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+#include "mococrw/hsm.h"
+
+namespace mococrw
+{
+using namespace openssl;
+
+const std::string &HSM::getName() { return _name; }
```

Comment:
I still don't see a use-case for this _name attribute but I guess we'll see that in the next PRs.

Reply:
This is not a vital component and not to be given significant importance. It is more with the intention of having a descriptive name, as a string, associated with concrete HSM classes (e.g., for logging).